### PR TITLE
Adding get_events method (api_service)

### PR DESCRIPTION
**Pull request by msotelo-eb**

### CODE REVIEW

File: search_events_app/models/event.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class Event:
-    def __init__(self, name, url, country=None, feature=[]):
+
+    def __init__(self, name, url, country=None, feature=[], **kwars):
```
Comment (mmonzo-eb) [inline search_events_app/models/event.py:3]: Name the parameter "kwargs"

File: search_events_app/services/api_service.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from search_events_app.models.event import Event
+from search_events_app.services.dummy_api import DummyApi
+
+
+def get_events():
+
```
Comment (mmonzo-eb) [inline search_events_app/services/api_service.py:6]: remove this blank line

File: search_events_app/models/event.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 class Event:
-    def __init__(self, name, url, country=None, feature=[]):
+
+    def __init__(self, name, url, country=None, feature=[], **kwargs):
```
Comment (arielaranda-eb) [inline search_events_app/models/event.py:3]: never put a mutable as a default argument, should be `feature=None` and then:
`if not feature:
    feature = []`
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

File: search_events_app/tests.py [hunk 4]
```diff
@@ -36,3 +40,38 @@ def test_event_model(self):
...
+                'features': ['Repeating event'],
+                'language': 'English',
+                'category': 'Fashion',
+                'format': 'Expo',
+            },
+        ]
+    )
+    def test_get_events(self, mock_get):
+        result = api_service.get_events()
+        self.assertTrue(isinstance(result[0], Event))
```
Comment (ravila-eb) [inline search_events_app/tests.py:76]: You could use self.assertIsInstance() 
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsInstance

File: search_events_app/tests.py [hunk 5]
```diff
@@ -36,3 +40,38 @@ def test_event_model(self):
...
+                'organizer_name': 'Fashion Revolution',
+                'country': '',
+                'url': 'https://www.eventbrite.co.uk/e/fashion-question-time-tickets-90925824589?aff=ebdssbonlinesearch',
+                'start_date': '2021-01-30',
+                'features': ['Repeating event'],
+                'language': 'English',
+                'category': 'Fashion',
+                'format': 'Expo',
+            },
+        ]
```
Comment (ravila-eb) [inline search_events_app/tests.py:72]: Maybe, this mock (and the next ones) should be in a different file and imported from there, just to keep a clear view of the tests.
What do you think about this?

Comment (msotelo-eb) [reply search_events_app/tests.py:72]: We thought the same thing but we didn't know how to do that. Thanks Romi

**Pull request merged**